src/svc.rs

```diff
@@ -103,6 +104,14 @@ impl<S> Stack<S> {
         self.push(TimeoutLayer::new(timeout))
     }
 
+    /// Validates that this stack serves T-typed targets.
+    pub fn makes<T>(self) -> Self
```

Comment:
unused in this PR?

Reply:
yeah, but it's useful. this module is for useful things when working with stacks.